### ISSUE TITLE
Automatically detect CSV separator

### ISSUE THREAD

**Issue by jhogervorst** [wijmo/gc-excelviewer#3]
Thanks for making this extension! I use it quite often and it's very useful, especially since it handles large files well 🙌

The only annoyance I have is that I manually need to update the settings to switch between the `,` and `;` CSV separator. I often handle both kinds of files, so many times I need to switch between those options.

It would be nice if the extension could automatically detect (or guess) the separator that's being used, and apply that.

**Comment by nielsvdc** [comment]
Yes, automatically detecting the separator would be a useful addition. Or the possibility to change the separator while the file is in preview, without having to change the settings.
